index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width" />
+  <link href="https://fonts.googleapis.com/css?family=Pacifico&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css?family=Varela+Round&display=swap" rel="stylesheet" />
+  <link rel="icon" href="/images/favicon.ico" />
+  <link rel="stylesheet" type="text/css" href="style.css" />
+  <title>Calendar Project</title>
+</head>
+
+<body>
+  <section>
+    <div class="image-box">
+      <img src="images/vanilla_coding_logo.png" />
+    </div>
+    <h1>Calendar</h1>
+
+    <!-- 달력 Start -->
+    <div class="calendar-container">
+      <div class="current-day">
+        <p></p>
+        <p></p>
+      </div>
+      <p class="current-month"></p>
+      <div class="calendar">
+        <table>
+          <thead>
+            <tr>
+              <th>SUN</th>
+              <th>MON</th>
+              <th>TUE</th>
+              <th>WED</th>
+              <th>THU</th>
+              <th>FRI</th>
+              <th>SAT</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+            </tr>
+            <tr>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+            </tr>
+            <tr>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+            </tr>
+            <tr>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+            </tr>
+            <tr>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+            </tr>
+            <tr>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+            </tr>
+          </tbody>
+        </table> <button class="button prev">
+          <<</button> <button class="button next">>></button>
```

Comment:
아래와 같이 줄바꿈을 하여 표현하는 것이 좋습니다.
```suggestion
        </table> 
        <button class="button prev"></button> 
        <button class="button next"></button>
```

Reply:
바쁘신 와중에 신경 써서 코드 리뷰 해주신 점 감사 드립니다.!!
알려 주신 부분들 보완 해서 다음 과제 화이팅 하겠습니다. 좋은 한주 되세요!